packages/smart-wallet/src/smartWallet.js

```diff
@@ -232,6 +232,16 @@ export const prepareSmartWallet = (baggage, shared) => {
     return store;
   });
 
+  const getPurseForBrand = (brand, { bank, invitationPurse, helper }) => {
```

Comment:
We're losing some static typing here, right? How about adding type annotations?

_This is one thing where I wish `tsc` were as nice as `flow`: with `flow`, factoring out a local function doesn't lose static type info._

Reply:
I'm unclear on what inference TypeScript does vs. does not do, but at any rate this was straightforward. Done.